src/components/Categories.vue

```diff
@@ -30,6 +20,13 @@ export default class Categories extends Vue {
   onSelect(category: Category) {
     return category;
   }
+
+  render():VNode {
+    return createElement('div', { id: 'Categories' }, this.categories.map((category, index) => (
+      createElement('div', { title: category.label, key: index, class:['category', { active: category.name === this.current }],
```

Comment:
Did you replace the use of templates here?

Reply:
yes, i did; because the v-html you used cannot pass github sonarCloud check. also, render function is more effective then temlate,  so i replace it with render function.